### PR TITLE
added BikeDockingStation data model

### PR DESCRIPTION
**Pull request by shyam28598**

### CODE REVIEW

**Comment by albertoabellagarcia** [comment]
These attributes are equivalent to the existing one (they are defined as integers instead of numbers)
- totalSlotNumber
- availableBikeNumber
- freeSlotNumber
- outOfServiceSlotNumber

These attributes are different and therefore there should be added
- mediaURL (there are a share attribute image and seeAlso in commons)
- stationCode
- agency_url
- agency_name

These attributes  are already by default
- stationName (the 'name' attribute seems to be the equivalent.)
- observationDateTime use to be used dateObserved

so it would be possible to extend current data model.

**Comment by shyam28598** [comment]
Hey Alberto I have added a new Data model folder WmgmtBin which is a data model for solid waste management bins. Please take a look at it and let me know if there are any changes that can be made

**Comment by albertoabellagarcia** [comment]
Regarding the BikeDockingStation, minor comments
- naming convention used to be camelCase (unless a regulation prevent doing it) There are several properties name xxx_yyy, it would be right to use xxxYyy
- ObservationDatetime is a date (the model is wrong)
- mediaUrl could be "format": "uri"  ??
- $ref1 and $ref2 belong to the template, right would be both $ref

**Comment by albertoabellagarcia** [comment]
Regarding wmgmtBin, I'd rather prefer WasteManagementBin, but if implemented it could be kept
- $ref1 and $ref2 (fix to $ref)
- binClearedTime and binLoggedTime model should be DateTime, isn't it?
- camelCase properties to license_plate
- XXXXId is prefered to XXXXID (binID, wardID) . Are they relationships? (Then it should be changed the type) and in the description of relationship

-

**Comment by albertoabellagarcia** [comment]
example.jsonld for management appears to be binary, so I cannot review it.

**Comment by albertoabellagarcia** [comment]
Type of entity should be BikeDockingStation (without iudx:), while in the @context could be iudx:BikeDockingStation 
I have doubts with the id of the example.jsonld from BikeDockingStation

**Comment by shyam28598** [comment]
Hey Alberto, The properties that are xxx_yyy are borrowed properties from Google's GTFS, hence why we have not used Camel case. Regarding the date time properties I have used the format that is given in the SmartDataModels template for date-time objects. I have changed the format for the url, regarding the naming conventions maybe Mukunth or Abhay will be able to explain better. I will make the rest of the changes and push them.

File: WmgmtBin/schema.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          "description": "Property. Model:'https://schema.org/Text'. Color of the bin. Could be used for indicating the type of waste. The color coding should follow the conventions applicable to the geographical area the bins are located."
+        },
+        "binClearedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Property. Model:'https://schema.org/Text'. Time when the bin was last cleared."
+        },
+        "wardID": {
+          "type": "string",
+          "description": "Property. Model:'https://schema.org/Text. Ward ID of the entity corresponding to this observation."
```
Comment (albertoabellagarcia) [inline WmgmtBin/schema.json:43]: warId?
is it a relationship?

File: WmgmtBin/schema.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        },
+        "binFullnessThreshold": {
+          "type": "number",
+          "description": "Property. Model:'https://schema.org/Number'. The fullness threshold level of the bin defined as the level (in terms of percentage) when the bin full alert or notification will be generated."
+        },
+        "binRecommendedLoad": {
+          "type": "number",
+          "description": "Property. Model:'https://schema.org/Number'. Recommended load (weight) that the waste bin corresponding to this observation can hold."
+        },
+        "binID": {
```
Comment (albertoabellagarcia) [inline WmgmtBin/schema.json:69]: binId? is it a relationship?

File: BikeDockingStation/schema.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (albertoabellagarcia) [inline BikeDockingStation/schema.json:1]: see https://github.com/smart-data-models/dataModel.Transportation/blob/master/BikeHireDockingStation/schema.json for potential merge

File: WmgmtBin/schema.json [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  ],
+  "allOf": [
+    {
+      "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/GSMA-Commons"
+    },
+    {
+      "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+    },
+    {
+      "properties": {
```
Comment (albertoabellagarcia) [inline WmgmtBin/schema.json:20]: missing entity type?

File: WmgmtBin/schema.json (hunk 1 above)
Comment (shyam28598) [reply WmgmtBin/schema.json:43]: wardID is a number assigned to a ward (section of a city). It is not considered a relationship

File: WmgmtBin/schema.json (hunk 2 above)
Comment (shyam28598) [reply WmgmtBin/schema.json:69]: binID is  a number assigned to a waste carrying bin. It is also not considered a relationship

**Pull request merged**